src/components/Note/EditableNoteMentions.tsx

```diff
@@ -0,0 +1,187 @@
+import uniqueId from 'lodash.uniqueid';
+import React, { FC, useEffect, useState } from 'react';
+import Tribute, { TributeItem } from 'tributejs';
+import Button from '../Button/Button';
+import ButtonToolbar from '../Button/ButtonToolbar';
+import Card from '../Card/Card';
+import CardBody from '../Card/CardBody';
+import FormLabelGroup from '../Form/FormLabelGroup';
+import Input from '../Input/Input';
+import { Note } from './Note.types';
+import NoteHeader from './NoteHeader';
+
+type MentionableUser = {
+  key: string;
+  value: string;
+  email: string;
+};
+
+function injectTribute(mentionableUsers: MentionableUser[], noteBodyId: string) {
+  if (mentionableUsers) {
```

Comment:
Seems like `mentionableUsers` will default to `[]`, so this should always be true. Did you mean to inspect `mentionableUsers.length`?